lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -350,14 +385,51 @@ class AwsCompileCloudFrontEvents {
 
             let behavior = {
               ViewerProtocolPolicy: 'allow-all',
-              ForwardedValues: {
-                QueryString: false,
-              },
             };
+            let shouldAssignCachePolicy = true;
             if (event.cloudFront.behavior) {
+              if (
+                event.cloudFront.behavior.ForwardedValues ||
+                event.cloudFront.behavior.MinTTL ||
+                event.cloudFront.behavior.MaxTTL ||
+                event.cloudFront.behavior.DefaultTTL
+              ) {
+                this.serverless._logDeprecation(
+                  'CLOUDFRONT_CACHE_BEHAVIOR_FORWARDED_VALUES_AND_TTL',
+                  'Cloudfront has deprecated the use of the ForwardedValues, MinTTL, MaxTTL and DefaultTTL field to configure cache behavior. Please use `provider.cloudfront.cachePolicies` to define Cache Policies and reference it here with `cachePolicy.name` property. You can also reference existing policies with `cachePolicy.id`.'
```

Comment:
In CLI logs, we usually wrap with `"` a programming names, it looks better than back tick. We can also use new lines to improve log readability